daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -916,13 +918,16 @@ func TestDryrunRequestJSON(t *testing.T) {
 	require.Equal(t, 1, len(dr.Accounts))
 	require.Equal(t, 1, len(dr.Apps))
 
-	var proto config.ConsensusParams
+	// var proto config.ConsensusParams
 	var response generated.DryrunResponse
 
-	proto.LogicSigVersion = 2
-	proto.LogicSigMaxCost = 1000
+	// proto.LogicSigVersion = 2
+	// proto.LogicSigMaxCost = 1000
+
+	// config.Consensus[dryrunProtoVersion] = proto
```

Comment:
delete commented out lines if they're not needed anymore.

Reply:
done